webapp/templates/home.html

```diff
@@ -27,7 +27,7 @@
                         
                         <h2></h2>
                         <div id="aeont1maa" class="card">
-                            <h3><img src="/static/img/pc.png"> Player Councillor Vote</h3>
+                            <h3><img src="/static/img/pc.png"> Player Councilor Vote</h3>
                             <p>The Council of Setons is a group of FAF contributors selected and tasked with the responsibility to guide the FAF community project.<br><br>
 							The Players Councilor is a member of the Council of Setons, who will be elected by the community to work on These Responsibilities:
```

Comment:
These Responsibilities -> these responsibilities

typos after that:
Communicate with as many FAF players as is possible
-> Communicate with as many FAF players as possible